pkg/virtctl/portforward/portforwarder.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net"
 	"strings"
 
-	"github.com/golang/glog"
+	"kubevirt.io/client-go/log"
 
```

Comment:
nit: Remove newline

Reply:
`go fmt` doesn't change anything.
do you have any formatting rules to follow?